src/main/c/cext/util.c

```diff
@@ -0,0 +1,7 @@
+#include <truffleruby-impl.h>
+
+double ruby_strtod(const char *nptr, char **endptr) {
+  #undef strtod
+  return strtod(nptr, endptr);
+  #define strtod(s,e) ruby_strtod((s),(e))    
+}
```

Comment:
Could you move this to `numeric.c` where similar functions live?

There is a comment there:
```
// Conversions between numeric types and from/to String
```

I'd like to avoid a `util.c` because I think it would mix many unrelated things longer term.

I see the function is declared in `util.h`, but I didn't use the header naming for our `.c` (and rather clarify with a comment what should go where) as e.g. `intern.h` contains a mix of many unrelated things.